src/openrct2/title/TitleSequence.cpp

```diff
@@ -168,20 +152,27 @@ void TitleSequenceCloseParkHandle(TitleSequenceParkHandle* handle)
     }
 }
 
-bool TitleSequenceSave(TitleSequence* seq)
+void TitleSequence::SwapConsecutiveCommands(size_t index)
+{
+    TitleCommand* first = &Commands[index];
+    TitleCommand* second = &Commands[index + 1];
+    std::swap(first, second);
```

Comment:
```suggestion
    assert(Commands.size() - 1 >= index + 1);
    if (Commands.size() - 1 >= index + 1)
    {
        TitleCommand* first = &Commands[index];
        TitleCommand* second = &Commands[index + 1];
        std::swap(first, second);
    }
```
We should add an assertion and if here to make sure no one tries to access the vector out of bounds.

(This is on commit https://github.com/OpenRCT2/OpenRCT2/pull/12815/commits/b0f7e8d558e04b4736b41257400ceab4cd73a38b)